src/ui/popup.js

```diff
@@ -231,6 +234,22 @@ export default class Popup extends Evented {
         return this.setDOMContent(frag);
     }
 
+    /**
+     * Sets the popup's max width.
+     *
+     * @param pixel A string representing the pixel value for the maximum width.
+     * @returns {Popup} `this`
+     */
+    getMaxWidth() {
+      return this._container;
+    }
+
+    setMaxWidth(maxWidth: string) { // should this be number?
```

Comment:
I think it makes sense to keep this as a string since it's setting a CSS property which takes a string. I could envision taking a number as input and converting it to a string but this property doesn't have to be in pixels. It could be ems, rems, points or any other valid CSS unit of measure and we wouldn't know what the user intended if they supplied a number. It may be reasonable to default to pixels, but I think it's clearer to be explicit and let the user define the units as part of the input.